### PR TITLE
 Added a 10 x 10 level

### PR DESCRIPTION
**Pull request by 0x48piraj**
## Things Done : 

* Added a 10 x 10 level to make the game more enjoyable :heart:

### CODE REVIEW

File: js/levels.js [hunk 1]
```diff
@@ -19,7 +19,16 @@ function createTestLevel()
...
+    [NullTile, door.none, door.left, door.right, door.none, door.right, door.none, door.none, door.right, door.top],
+    [door.none, door.right, door.right, door.right, door.goal, door.none, door.top, door.left, door.right,],
+    [door.none, door.top, door.both, door.both, door.wheel, door.top, NullTile, door.right, door.top, door.left],
+    [door.none, door.top, PlayerStartsAt(door.black), door.both, door.both, door.top]
+    [door.none, door.top, door.both, door.both, door.both, door.top, door.top, door.left, door.right],
+    [door.none, door.top, door.wheel, door.both, door.both, door.wheel, door.right, door.top, door.top],
+    [door.none, door.top, door.both, door.wheel, door.both, door.top, door.none, door.right, door.top, door.both],
+    [door.none, door.none, door.none, door.none, door.none, door.none, door.both, door.right, door.top, door.none],
+    [door.left, door.top, door.,none, door.,right, door.none, door.wheel, door.none, door.none, door.none],
+    [NullTile, door.none, door.none, door.none, door.none, door.none, door.,left, door.top, door.right, door.top]
```
Comment (niccokunzmann) [inline js/levels.js]: door.,left - could you add a preview link and show it is working?

Comment (0x48piraj) [reply js/levels.js]: Oopsy. will correct that!
And Sure!

**Comment by 0x48piraj** [comment]
Ping @niccokunzmann :heart_decoration: 

> **_Here's a working Test-Link : http://rawgit.com/0x48piraj/labyrinth/level/index.html :100:_**

**Comment by niccokunzmann** [comment]
Thanks! The code is not reachable.. but I think you have something in mind.

**Pull request merged**